platform/util/platform-util/pom.xml

```diff
@@ -49,10 +49,48 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.jodah</groupId>
+            <artifactId>failsafe</artifactId>
+            <version>${jodah-failsafe.version}</version>
+        </dependency>
     </dependencies>
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
platform-util is jar, not a bundle. Shading was the recommended approach. But if you want, we can talk about in person. Maybe there's a better way?